dlrover/python/training_event/exporter.py

```diff
@@ -0,0 +1,315 @@
+# Copyright 2025 The DLRover Authors. All rights reserved.
```

Comment:
Need a doc(define in doc/design) for this feature.

- the background
- the introduction of all the exporter components
- predefined event(in table format)
- how to extend a new type event